app/templates/components/modals/tax-info-modal.hbs

```diff
@@ -19,7 +19,7 @@
           {{/each}}
         </div>
       </UiDropdown>
-    </div>
+    </form>
```

Comment:
Repeated. Put in another component

Reply:
ok and is it working now?

Reply:
autocomplete="disable" was working as well. This is a workaround. Now it is not showing error when country is required

Reply:
inside input tag? ```autocomplete="disable"```

Reply:
@iamareebjamal ```autocomplete="disable"``` is not working when I am using it on input tag

Reply:
Then just move this in a component